SERVER/static/js/chat.js

```diff
@@ -21,20 +21,36 @@
   };
 
   sendMessage = function() {
```

Comment:
i don't see another way to do this. send_message grabs the message that the user types; puts it in a json object with the date and the username and sends it to the server. If you'd rather me put the code that's putting it in a packet in a different function then i'll do that.